redaxo/src/addons/mediapool/lib/var_media.php

```diff
@@ -16,7 +16,16 @@ class rex_var_media extends rex_var
     protected function getOutput()
     {
         $id = $this->getArg('id', 0, true);
-        if (!in_array($this->getContext(), ['module', 'action']) || !is_numeric($id) || $id < 1 || $id > 10) {
+        if (!in_array($this->getContext(), ['module', 'action'])) {
+            return false;
+        }
+        if (!is_numeric($id)) {
+            return false;
+        }
+        if ($id < 1) {
+            return false;
+        }
+        if ($id > 10) {
```

Comment:
Gerade bei solchen Range-Bedingungen ("kleiner als X oder größer als Y") finde ich es nicht sinnvoll, die in mehrere Early-Returns aufzuteilen.
Hatten wir ja letztens auch schon mal, als du mit den Early-Returns begonnen hast.
Das Tool kann natürlich nicht für uns Menschen logisch zusammengehörende Bedingungen erkennen.

Oder anders gesagt: Ich bin wie gesagt großer Freund von Early-Returns. Aber an vielen Stellen waren es auch vorher schon Early-Returns und sie wurden für mein Empfinden unnötig auseinander gezogen in mehrere Returns.

Gleichzeitig gibt es aber natürlich auch Stellen, wo das Auseinanderziehen sinnvoll ist, weil die Bedinungen vorher zu lang und zu unlesbar war. Also das Trennen von dem Context-Check hier vom id-Check ist zum Beispiel natürlich gut.